evm/src/main/java/org/hyperledger/besu/evm/tracing/StandardJsonTracer.java

```diff
@@ -174,6 +175,10 @@ public void tracePostExecution(
     }
     final int opcode = currentOp.getOpcode();
     final Bytes returnData = messageFrame.getReturnData();
+    long thisGasCost = executeResult.getGasCost();
+    if (currentOp instanceof AbstractCallOperation) {
+      thisGasCost += messageFrame.getMessageFrameStack().getFirst().getRemainingGas();
+    }
```

Comment:
@shemnon  Should we do the same change in `DebugOperationTracer`?
```
            operationResult.getGasCost() == 0
                ? OptionalLong.empty()
                : OptionalLong.of(operationResult.getGasCost()),
```

Reply:
It required a bit more blocking and tacking to make sure it didn't leak into the `trace_` series of calls, which do actual consumed gas separately.  However it is done.